Update quantity.md -clarification of quantity expectations. (#1159)
spdx/spdx-3-model: model/Core/Properties/quantity.md

## Description

Quantity is a property that defines the count, multitude or magnitude.

> Express it as an xsd:decimal when possible. If it is not an xsd:decimal, place the notation that is being used in the parent element comment field.

Reviewer: ```suggestion

Express it as an xsd:decimal whenever possible.
If it is not an xsd:decimal, place the notation being used in the parent element's comment field.
```

put 's after element, and shorten the second sentence a bit.
